cmd/plugin.go

```diff
@@ -82,12 +82,16 @@ Examples:
   steampipe plugin install aws
 
   # Install a specific plugin version
-  steampipe plugin install turbot/azure@0.1.0`,
+  steampipe plugin install turbot/azure@0.1.0
+
+  # Install a plugin with silent/quiet progress indicator
+  steampipe plugin install --silent aws`,
 	}
 
 	cmdconfig.
 		OnCmd(cmd).
-		AddBoolFlag(constants.ArgHelp, false, "Help for plugin install", cmdconfig.FlagOptions.WithShortHand("h"))
+		AddBoolFlag(constants.ArgHelp, false, "Help for plugin install", cmdconfig.FlagOptions.WithShortHand("h")).
+		AddBoolFlag(constants.ArgSilent, false, "No installation progress indicator", cmdconfig.FlagOptions.WithShortHand("s"))
```

Comment:
Since we are only disabling the progress bars I think this should be `--progress` and not `--silent`. This is also inline with the `check` command which has a similar funcitonality

https://github.com/turbot/steampipe/blob/f47985f88fc111d267640afec062e67f09d62efa/cmd/check.go#L68